packages/xstate-wallet/src/workflows/application.ts

```diff
@@ -71,14 +72,20 @@ export interface WorkflowServices extends Record<string, ServiceConfig<WorkflowC
     context: ChannelIdExists
   ) => StateMachine<ConcludeChannel.Init, any, any, any>;
 
+  invokeChallengingProtocol: (
+    context: ChannelIdExists
+  ) => StateMachine<ChallengeChannel.Init, any, any, any>; // FIXME: add type
```

Comment:
Should I be typing any of these `any`s? @andrewgordstewart

Reply:
I don't personally think the `WorkflowServices` type adds a lot of value.